src/pkg/dns/resolver.go

```diff
@@ -38,7 +38,7 @@ var rootServers = []*net.NS{
 }
 
 func (r RootResolver) LookupIPAddr(ctx context.Context, domain string) ([]net.IPAddr, error) {
-	for i := 0; i < 10; i++ {
+	for range [10]int{} {
```

Comment:
This can be `[10]struct{}{}` cf. the others, but I actually don't mind the C `for`.